src/System Application/App/Entitlements/DelegatedBCAdminAgentPartner.Entitlement.al

```diff
@@ -0,0 +1,28 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Security.AccessControl;
+
+using System.Azure.Identity;
+using System.Environment.Configuration;
+using System.Email;
+using System.Apps;
+
+entitlement "Delegated BC Admin agent - Partner"
+{
+    Type = Role;
+    RoleType = Delegated;
+    Id = '00000000-0000-0000-0000-000000000010';
```

Comment:
Just making sure this is the right Id?

Reply:
Yes, this is the id for the delegated role.